.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "privaterelay/locales"]
 	path = privaterelay/locales
 	url = https://github.com/mozilla-l10n/fx-private-relay-l10n.git
+[submodule "static/images/material-design-icons"]
```

Comment:
It worked for me while the server was running, but then eventually it crashed too and I couldn't restart it.

However, with some more searching I did find out that there is an npm package published by Google as well. It doesn't include the outlined version, so it looks slightly different than designed, but at least it's clear where the image came from and it doesn't crash the server.

Reply:
Hmm, I thought it might be a temporary and local glitch, but `npm install` takes ages now, and appears to hit the CircleCI timeout too. The two remaining options I can think of are:

1. Increase the CircleCI timeout and live with a slow `npm install`.
2. Live with vendoring the image.

I don't have a particular preference - you?